eq-author/src/App/page/Routing/RoutingPage/RoutingEditor/RuleEditor/index.js

```diff
@@ -55,6 +55,7 @@ const SmallSelect = styled(Select)`
   display: inline-block;
   width: auto;
   margin: 0 0.5em;
+  line-height: 1.5;
```

Comment:
1.5 is a little big? Could we bring it down to 1.25, should still do the trick.

Reply:
Good suggestion. Changed it to 1.25.